autoload/SpaceVim/layers/chinese.vim

```diff
@@ -20,7 +21,8 @@ endfunction
 
 function! SpaceVim#layers#chinese#config() abort
   let g:_spacevim_mappings_space.x.g = {'name' : '+translate'}
-  call SpaceVim#mapping#space#def('nnoremap', ['x', 'g', 't'], 'Ydc', 'translate current word', 1)
+  call SpaceVim#mapping#space#def('nnoremap', ['x', 'g', 't'], 'Ydc'         , 'translate current word'  , 1)
+  call SpaceVim#mapping#space#def('nnoremap', ['l', 'c']     , 'CheckChinese', 'Check with ChineseLinter', 1)
```

Comment:
`SPC l` prefix is for program language.

Reply:
But `:CheckChinese` is only for Chinese, `*.md *.markdown *.text *.txt *.rst *.cnx *.wiki`, etc.

Reply:
hmm, I will merged it locally, and fix the key binding.